models/boards.js

```diff
@@ -1274,6 +1262,23 @@ function boardRemover(userId, doc) {
   );
 }
 
+Boards.uniqueTitle = title => {
+  const m = title.match(/^(?<title>.*?)\s*(\[(?<num>\d+)]\s*$|\s*$)/);
+  const base = escapeForRegex(m.groups.title);
+  let num = 0;
+  Boards.find({ title: new RegExp(`^${base}\\s*\\[\\d+]\\s*$`) }).forEach(
+    board => {
+      const m = board.title.match(/^(?<title>.*?)\s*\[(?<num>\d+)]\s*$/);
```

Comment:
Please add backtick quotes to regex string, before starting `/` and after ending `/`